pkg/handlers/admin_entities_test.go

```diff
@@ -34,7 +34,7 @@ var testConfig = config.AgentConfig{
 	Name:    "3",
 }
 
-func TestHealthHandlerBothServicesStarted(t *testing.T) {
+func TestAdminHealthHandlerBothServicesStarted(t *testing.T) {
```

Comment:
Why renamed it?

Reply:
since a new test with a similar name was added here: https://github.com/optimizely/agent/pull/232/files#diff-06adbb118deb5fc1de8da69c3b3668b6R29